tools/run_tests/xds_k8s_test_driver/framework/infrastructure/k8s.py

```diff
@@ -573,3 +573,16 @@ def _replicas_available(cls, deployment: V1Deployment, count: int) -> bool:
         return (isinstance(deployment, V1Deployment) and
                 deployment.status.available_replicas is not None and
                 deployment.status.available_replicas >= count)
+
+
+def get_pod_restarts(k8s_namespace: KubernetesNamespace,
```

Comment:
Let's make it a part of the class, same as other methods. I know it's not perfect, but we should refactor the whole thing anyway. At least we'll be consistent in being imperfect.

Reply:
Originally I did put it into the class, but got some complains from sanity tests, let me try again.